docs/example-migration-v5-node-template-to-v6.md

```diff
@@ -135,53 +135,35 @@ This file needs to be updated to use the `shopify` api instance as well as the n
  }
 ```
 
-### 6. Copy the `SQLiteSessionStorage` class from the `@shopify/shopify-app-express` package
+### 6. Install the `@shopify/shopify-app-session-storage-sqlite` package
```

Comment:
This is not for this line but in Point 4, you mention `Shopify.Context` but for v6, this is replaced by `Shopify.config` if I made no mistake ?

Reply:
I've left it as-is (I added "v5" just before the mention to help clarify), because the instruction is to take the configuration in the current template, the current v5 `Shopify.Context` configuration, and move it into the `shopify.js` file being created.  I think I still need to refer to `Shopify.Context` as that's the migration part.